_summaries/hendrycks2016gaussian_2.md

```diff
@@ -1,19 +1,42 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
+title: Gaussian Error Linear Units (GELUs), Hendrycks, Gimpel Summary
+paper: {{hendrycks2016gaussian}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: jchavezberkeley
+score: 8
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+This paper proposes the Gaussian Error Linear Unit (GELU), a neural activation function that performs well against ReLU and ELU in similar training experiments. It is often the case that a "deterministic decision" is wanted from a neural network, which motivates the need for a new nonlinearity.
```

Comment:
Maybe explain the motivation a bit more? The term "deterministic decision"  may need some context